8.6/js/firstjs.js

```diff
@@ -0,0 +1,12 @@
+
+var text = 'Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.';
+var green = ('Zielone słonie').toUpperCase();
+var animal = 'Hieny';
+
+var text2 = text.replace('Papugi', green);
+
+console.log(text2.length);
+console.log(text2.slice(0,46));
```

Comment:
`46` musisz wyciągnąć z tego stringa - bo obecnie jest to strasznie na sztywno. Użyj właściwości `length`